src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -26,6 +26,12 @@ private static string GetFileNameFromResourceName(string s)
 
         public int CreateEmptyProject(string languageName, string templateDir)
         {
+            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "project.json")))
+            {
+                Reporter.Error.WriteLine($"Creating new {languageName} project failed, project.json already exists.");
```

Comment:
A better check would be to see if any of the extracted zip entries is already there in the folder, for which it will fail..